renovate.json

```
@@ -44,7 +44,14 @@
     "stacks/go/1.0.2/**",
     "stacks/go/1.1.0/**",
     "stacks/go/2.0.0/**",
-    "stacks/go/2.1.0/**"
+    "stacks/go/2.1.0/**",
+    "stacks/nodejs/2.1.1/**",
+    "stacks/nodejs-angular/2.0.2/**",
+    "stacks/nodejs-nextjs/1.0.3/**",
+    "stacks/nodejs-nuxtjs/1.0.3/**",
+    "stacks/nodejs-react/2.0.2/**",
+    "stacks/nodejs-svelte/1.0.3/**",
+    "stacks/nodejs-vue/1.0.3/**"
```

Think a typo in the intended `nodejs-vue` stack version:
```suggestion
    "stacks/nodejs-vue/1.0.2/**"
```

---

Oops my bad, I'll correct that real quick thank you!